core/src/main/scala/org/broadinstitute/dsde/rawls/util/SentryEventFilter.scala

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.dsde.rawls.util
+
+import io.sentry.SentryEvent
+
+object SentryEventFilter {
+  /**
+   * Client side filtering for spammy Sentry events
+   */
+  def filterEvent(event: SentryEvent): SentryEvent = {
+    val msg = Option(event.getMessage)
+    if (msg.isDefined) {
+      val actualMsg = msg.get.getMessage
+      if (actualMsg.contains("requirement failed: count cannot be decreased") ||
+        actualMsg.contains("pet service account not found")) {
+        return null
+      }
+    }
+
+    event
```

Comment:
totally for style, your call on what to do - if you wanted to be more Scala-y:
```suggestion
    Option(event.getMessage) match {
      case Some(msg) if msg.getMessage.contains("requirement failed: count cannot be decreased") ||
        msg.getMessage.contains("pet service account not found") => null
      case _ => event
    }
```